packages/bot-web-ui/src/components/quick-strategy/selects/__tests__/symbol.spec.tsx

```diff
@@ -90,24 +91,36 @@ describe('<SymbolSelect />', () => {
             </StoreProvider>
         );
     });
-    it('should render SymbolSelect', () => {
+    it('Should render SymbolSelect', () => {
```

Comment:
@maryia-matskevich-deriv even though it's not an issue, but as everywhere else we are using it with small letters so I am willing to know if there is any specific reason for capitalizing the first letter here.

Reply:
✅done